### ISSUE TITLE
About client v1

### ISSUE THREAD

**Issue by Rain0Ash** [NL0bP/Archeage-Server-emulator#8]
Download client v1
http://megaoblako.ru/games/456-archeage-l-rus-rus-2013-10-rossiya-klient-ot-150214.html 
https://cloud.mail.ru/public/cdaf4fb889b9/Archeage_enotbox.com.rar
compact.sqlite3 v1
https://drive.google.com/file/d/1stbg2-BUbp936ZdRwnJ0kxGDOT34nxvd/view?usp=sharing
decrypted.data for base
https://drive.google.com/open?id=1cEBLl9-IMcNhaTBk9cBP3KULaQUC23_9
decrypted base
https://drive.google.com/open?id=1mIWfpiZxS1aDL1CO1xT5xfusBTjV4P5D
error when I try to launch client
https://games.mail.ru/support/aa/#/questions/1515
P.S. Who-thread wants music? 128 files 👍 🎧 😄 
![default](https://user-images.githubusercontent.com/30903358/44201308-99d38800-a151-11e8-94c8-d2f50e8ebfde.png)
Add to my collection

**Comment by sxeroot** [comment]
well dont know much about the security russian client used but that message sais that since 2017 date the gameguard is turned off i guess that client still tries to use GG and fails since prolly the licensing of it is expired.

as for the database its still just the client database unfortunately alot of usefull data but unless im missing it a bunch is still not there

**Comment by Rain0Ash** [comment]
I think we should run the client in any way without GG. I already forgot where it was, but I read that in some version there was also a server db in the client. For the first time the client started to run, but the computer decided to rest (maybe because of the client), after rebooting, it gives this error.

**Comment by DancerS** [comment]
archeage.exe -r +auth_ip 127.0.0.1:1237 -uid test -token 11111111111111111111111111111111111111111111111111 11111111111111 +acpxmk

**Comment by DancerS** [comment]
`archeage.exe -r +auth_ip 127.0.0.1:1237 -uid test -token 11111111111111111111111111111111111111111111111111 11111111111111 +acpxmk`

> +acpxmk - disable GameGuard
> -r - localization en
> +auth_ip - address and port of the server login
> -uid - account name
> -token - password (32 bytes) exactly in this form this password will come to the login.


**+acpxmk always put at the end of all parameters that the game will not crash.**

**Comment by Rain0Ash** [comment]
Thanks.
![default](https://user-images.githubusercontent.com/30903358/44206517-c348df80-a162-11e8-8ef6-7b767dab6a28.png)

**Comment by Rain0Ash** [comment]
Wait, that's another question. I have a very long (or infinite) movie without sound.
![default](https://user-images.githubusercontent.com/30903358/44206841-c8f2f500-a163-11e8-9347-86b081be2f8a.png)
![default](https://user-images.githubusercontent.com/30903358/44206934-0c4d6380-a164-11e8-96b7-2c5a0d02d1d8.png)
Either this screen saver when there is no connection with the server.

**Comment by JeyFrey** [comment]
@sxeroot 
Someone in this forum had the compact_server, so anyone russian speaking could ask there if anyone still has a backup.
https://forum.zone-game.info/showthread.php?t=33319&page=13

Guess my Google translator russian wont be enough.

**Comment by DancerS** [comment]
I asked, no

**Comment by DancerS** [comment]
I tried everything I already know, but I could not remove this error in the database. Maybe someone knows how to fix it?

**Comment by Rain0Ash** [comment]
Yes, the server base appeared in 1.2, in addition, I believe that this version is better. However, for all the time of searching I found only a couple of references to it, which are also much more likely not working. And two links to piratebay without distributors.
example
https://tapochek.net/viewtopic.php?t=154971
http://tas-ix.net/viewtopic.php?t=98543
account is required, account is not registered
http://gamesontorrent.ru/pc-torrents/8158-archeage-russkij-klient-distributiv-11-04-2014.html (maybe)
P.S. for the links of responsibility do not carry, that I see that and throw off. AT YOUR OWN RISK
I did not find anything else.

**Comment by JeyFrey** [comment]
Someone on RageZone pm'ed me about someone on Discord who has the 1.2 server database, we'll see if I can get anything.

**Comment by Rain0Ash** [comment]
[Offtop] To buy a pig in a poke. What kind of people are these, dog in the manger...

**Comment by JeyFrey** [comment]
Lets see, I messaged some people, called in some favors. And in the end you cant get scammed if you use PayPal. Will keep you guys updated. 
Btw I read more that one or two times now that people are looking for a discord channel for this project - anyone who is more experienced with Discord willing to create a channel?

**Comment by Rain0Ash** [comment]
@JeyFrey I can create Discord channel (technical and settings part)

**Comment by Rain0Ash** [comment]
Created. If anyone wants - join https://discord.gg/vn8E8E6

**Comment by jekadrost** [comment]
[https://cloud.mail.ru/public/Bfz9/QzUjBEqh8](url)

**Comment by nikes** [comment]
Login:
```
RegisterPacket(0x00, typeof(LoginAccept));
            RegisterPacket(0x03, typeof(AccountInfo));
            RegisterPacket(0x08, typeof(ServerList));
            RegisterPacket(0x0a, typeof(GameserverInfo));
            RegisterPacket(0x0c, typeof(LoginDenied));
```
Game:
```
RegisterPacket(0x00, 1, typeof(ClientConnected));
            RegisterPacket(0x05, 1, typeof(Dummy));
            RegisterPacket(0x06, 1, typeof(Dummy));
            RegisterPacket(0x37, 1, typeof(Dummy));
            RegisterPacket(0xcc, 1, typeof(Dummy));
            RegisterPacket(0x1b9, 1, typeof(Dummy));
            RegisterPacket(0x1bd, 1, typeof(HSRequest));
            RegisterPacket(0x1bc, 1, typeof(HGRequest));

            RegisterPacket(0x00, 2, typeof(ChangeState));
            RegisterPacket(0x0f, 2, typeof(Dummy));
            RegisterPacket(0x13, 2, typeof(Pong));
```
Stream:
```
 RegisterPacket(0x01, typeof(Join_Re));
            RegisterPacket(0x02, typeof(DoodadStream));
            RegisterPacket(0x03, typeof(DoodadsId));
```
[Packets.zip](https://github.com/NL0bP/Archeage-Server-emulator/files/2425598/Packets.zip)

It's my old sniffer xD

**Comment by nikes** [comment]
Here, u can get all packets s->c & c -> s; struct & opcodes, but need using IDA Pro,
https://mega.nz/#!mNg2XSTZ!WFKks72VQEj1GN7meIL8FnNtm94WQapSPyFSwV8X0EU

**Comment by nikes** [comment]
AA Client DataBase v 1.x+ + exe 
https://mega.nz/#!mBRH1A5C!tYwcqMsBUa1JSK07c8mar4-rFRGo2reei9JsL38Pjw4

**Comment by NL0bP** [comment]
@nikes Tell me, how did you work with coordinates, in particular consisting of 3 bytes? Sources can give?

**Comment by nikes** [comment]
@NLObP give me 3 bytes of cords x,y,z

**Comment by nikes** [comment]
@NL0bP xD, i found me old code....
        ```
public static float ConvertX(byte[] coords)
        {
            return (float)Math.Round(coords[0] * 0.002f + coords[1] * 0.5f + coords[2] * 128, 4, MidpointRounding.ToEven);
        }

        public static byte[] ConvertX(float x)
        {
            var coords = new byte[3];
            var temp = x;
            coords[2] = (byte)(temp / 128f);
            temp -= coords[2] * 128;
            coords[1] = (byte)(temp / 0.5f);
            temp -= coords[1] * 0.5f;
            coords[0] = (byte)(temp * 512);
            return coords;
        }

        public static float ConvertY(byte[] coords)
        {
            return (float)Math.Round(coords[0] * 0.002f + coords[1] * 0.5f + coords[2] * 128, 4, MidpointRounding.ToEven);
        }

        public static byte[] ConvertY(float y)
        {
            var coords = new byte[3];
            var temp = y;
            coords[2] = (byte)(temp / 128);
            temp -= coords[2] * 128;
            coords[1] = (byte)(temp / 0.5f);
            temp -= coords[1] * 0.5f;
            coords[0] = (byte)(temp * 512);
            return coords;
        }

        public static float ConvertZ(byte[] coords)
        {
            return (float)Math.Round(coords[0] * 0.001f + coords[1] * 0.2561f + coords[2] * 65.5625f - 100, 4, MidpointRounding.ToEven);
        }

        public static byte[] ConvertZ(float z)
        {
            var coords = new byte[3];
            var temp = z + 100;
            coords[2] = (byte)(temp / 65.5625f);
            temp -= coords[2] * 65.5625f;
            coords[1] = (byte)(temp / 0.2561);
            temp -= coords[1] * 0.2561f;
            coords[0] = (byte)(temp / 0.001);
            return coords;
        }

        public static float ConvertLongX(long x)
        {
            return ((x >> 32) / 4096f);
        }

        public static long ConvertLongX(float x)
        {
            return ((long)((x) * 4096)) << 32;
        }

        public static float ConvertLongY(long y)
        {
            return((y >> 32) / 4096f);
        }

        public static long ConvertLongY(float y)
        {
            return ((long)((y) * 4096)) << 32;
        }
```

Не мог бы ты дать мне доступ в Discord(Can u add me access to Discord)?

**Comment by NL0bP** [comment]
@nikes welcome
